sgRNA_design/sgRNA_finder.py

```diff
@@ -0,0 +1,98 @@
+from collections import defaultdict
+
+# Function to determine if a potential PAM site meets the specified range and 
+# efficiency values. pamLine should be one line of csv with format:
+# Start Pos,Stop Pos, sgRNA sequence, PAM sequence (NGG), strand, MIT score
+def pam_In_Range(pamLine,geneRange,effCutOff=0):
+
+	# Parse and split csv line into list of string
+	words = pamLine.strip("\n").strip(",").split(',')
+	# Calculate the end position for PAM sequence based on desired coverage
+	endRange = (geneRange[1] - geneRange[0])*geneRange[2] + geneRange[0]
+
+	# Can be called directly in if and return statements, but makes code
+	# difficult to read
+	pamSta = int(words[0]) # Start Position
+	pamStp = int(words[1]) # Stop Position
+	pamSeq = words[2]      # sgRNA sequence
+	pamNGG = words[3]      # PAM sequence
+	pamStr = int(words[4]) # plus or minus strand
+	pamEff = float(words[5]) # MIT efficiency score
+
```

Comment:
I would make the qualifiers more verbose (e.g. start, end, Target, NGG is ok, Sense, MITEfficiency